src/core/Table/Table.tsx

```diff
@@ -86,6 +87,20 @@ export type TableProps<
    * @default 300
    */
   intersectionMargin?: number;
+  /**
+   * A function that will be used for rendering a component for each row if that row is expanded.
+   * Component will be placed right after the row, in the same row group div.
+   */
+  expandedSubComponent?: (row: Row<T>) => ReactNode;
+  /**
```

Comment:
Not sure about the prop name.. But we can discuss this.

Reply:
Maybe just `subcomponent` (which is what react-table v6 called it) or `subRow`

Reply:
Renamed to `subComponent`

Reply:
I think expanded prefix I liked better.. That was concrete.. Cause subComponent could be also not necessarily the ex expanded content..Not sure.

Reply:
For me `subCompenent` is still not clear without description. Maybe `expansionComponent`, `expandedRow` or `expandedContent.`

Reply:
@mayank99 @veekeys can we help Julius with the naming here?

Reply:
I'm fine with `expandedContent` but I'm also fine with `subcomponent` because that's the term react-table uses

Reply:
It is currently renamed to subComponent